code/modules/urist/antagonist/conspiracies.dm

```diff
@@ -0,0 +1,105 @@
+//This file defines the various conspiracies
+//Unfortunately large amounts of boilerplate code, but cleaning it would require some major work
+//Begin terrible SS13-pun names, credit/blame goes to some ancient /tg/station thread:
+
+//////////////////////////BUILDABORGS///////////////////////////////
+
+var/datum/antagonist/agent/buildaborg/buildaborgs
+/datum/antagonist/agent/buildaborg/New()
+	..()
+	buildaborgs = src
+
+/datum/antagonist/agent/buildaborg
+	id = "Buildaborg"
+	role_text = "Buildaborg Group Leader"
+	role_text_plural = "Buildaborg Agents"
+	faction_role_text = "Buildaborg Agent"
+	faction_descriptor = "Buildaborg Group"
+	faction_verb = /mob/living/proc/convert_to_conspiracy
```

Comment:
in fact, it seems like that's what you intended to do, because the individual faction procs aren't actually used anywhere yet, you just use the basic conspiracy proc.